src/event.rs

```diff
@@ -196,6 +200,7 @@ pub struct Mouse {
     pub column: Column,
     pub cell_side: Side,
     pub lines_scrolled: f32,
+    pub last_press_pos: (usize, usize),
```

Comment:
I don't like the idea of using `last_press_pos` here to check that the URL has been clicked rather than dragged.

Though I'm not sure if there's any better solution for this problem.

Reply:
Maybe using a boolean to indicate that the mouse has moved since the last button down is a bit nicer. This could be updated in the mouse_moved method.

Checking that the cursor moved at least on cell seems reasonable. Also it would still allow single-cell selection to be recognized as a URL click.